lib/streamlit/runtime/caching/cache_resource_api.py

```diff
@@ -520,7 +519,7 @@ def read_result(self, key: str) -> CachedResult:
             return result
 
     @gather_metrics("_cache_resource_object")
-    def write_result(self, key: str, value: Any, messages: list[MsgData]) -> None:
+    def write_result(self, key: str, value: Any, messages: List[MsgData]) -> None:
```

Comment:
Do we want to replace `list`->`List` here? I thought we were moving toward using `list` in most places since it'll eventually be the normal way these types are declared, and we're able to in older Python versions by importing `annotations` from `__future__`.

Reply:
yep, makes sense 👍  I'm not even sure why I change that.